libs/category/src/selectors/category.selector.ts

```diff
@@ -175,7 +175,7 @@ export const selectSelectedCategory = createSelector(
 
 export const selectCategoryPageProducts = createSelector(
 	selectCategoryPageProductIds,
-	selectProductEntities,
+	getDaffProductEntitiesSelectors<DaffProductUnion>().selectProductEntities,
```

Comment:
I can't really do it here yet, because of how the category selectors are structured. Or at least I can't do it easily. Planning on changing it once the category selectors are changed too

Reply:
okay maker sense